src/index.ts

```diff
@@ -164,23 +161,23 @@ program
     )
     .option("-n, --note [note]", "Note attached to the release")
     .action(({ note }: any) => {
-        const isConfirmed = "Yes";
-        const isDenied = "No, let's do this release!";
+        const versions = blockDetails();
+        const { name } = versions;
         inquirer
             .prompt({
-                choices: [isConfirmed, isDenied],
-                message: "Are you releasing a breaking change?",
+                default: true,
+                message: `You are about to release your updates to ${name} to production.\nNon-major version changes will take effect immediately on the stores that have your block installed.\nContinue?`,
                 name: "releaseConfirmation",
-                type: "list",
+                type: "confirm",
             })
             .then((confirmation: any) => {
-                if (confirmation.releaseConfirmation === isConfirmed) {
+                if (confirmation.releaseConfirmation) {
+                    release(note);
+                } else {
                     log(
                         "If you are releasing a breaking change you should create a major release using publish -m"
```

Comment:
Maybe this should be updated to `using "element publish -m"` (with a second set of quotation marks.)